### ISSUE TITLE
[Last.fm] Unable to scrobble unknown artists even with regex setup

### ISSUE THREAD

**Issue by MinakoDev** [web-scrobbler/web-scrobbler#4360]
**Describe the bug**
The extension says that it scrobbles successfully and it shows that it works perfectly fine like any other song but it doesn't actually create it as a track on the Last.fm database so it doesn't work properly.

**How to reproduce**
1. I setup an regex which searches for an unknown artist (aka (.?)), then it'll get replaced as "Unknown Artist". I tried other artist names too like Artist and [unknown].
2. I go on a YouTube Shorts video and it says that it's scrobbling and it even says it's successfully scrobbled (I forced track recognition) but it doesnt create it as a track on the last.fm database and it doesnt show on my profile. Clicking on the track from the extension takes you to a 404 error page.

**Expected behavior**
Scrobble properly by creating a track with the specific artist from the regex on the Last.fm database if the artist's name is unknown and to show up on my profile especially as well.

**Screenshots**
https://images.joshuagmason.com/sharex/9r8od5.gif

**Debug logs**
https://images.joshuagmason.com/sharex/ekpnx6.png

**Environment (please complete the following information)**
-   OS: Windows
-   Browser: Opera GX
-   Extension version: Version 3.3.0

**Additional context**
Other songs with an already specified artist name works.

**Comment by RaylaLe100** [comment]
Came here to report this (or something very similar) too, regardless of regex config.

How it happens:
- listening to a music on youtube that doesn't yet have a lastfm page
- the extension marks the track as scrobbled with the green checkmark
- but the track is not scrobbled on lastfm

Workaround:
- simply listening to the track a second time triggers a real scrobble

That's really bothersome since I need to check on lastfm every track I listen to to make sure it doesn't fail to scrobble.

I can also say that this bug is fairly recent, probably not older than a week or two.

**Comment by MinakoDev** [comment]
> Came here to report this (or something very similar) too, regardless of regex config.
> 
> How it happens:
> 
> * listening to a music on youtube that doesn't yet have a lastfm page
> * the extension marks the track as scrobbled with the green checkmark
> * but the track is not scrobbled on lastfm
> 
> Workaround:
> 
> * simply listening to the track a second time triggers a real scrobble
> 
> That's really bothersome since I need to check on lastfm every track I listen to to make sure it doesn't fail to scrobble.
> 
> I can also say that this bug is fairly recent, probably not older than a week or two.

Normally if track recognition is forced, the extension would create the track on Last.fm if it can't find it then it would scrobble it the first time. Odd that.

**Comment by RaylaLe100** [comment]
> 
> Normally if track recognition is forced, the extension would create the track on Last.fm if it can't find it then it would scrobble it the first time. Odd that.
> 
> However, MusicBrainz work fine.

You're right, after further investigation it looks like my forced track recognition was turned off. Possibly a problem with my local memory during last update, but no longer a bug now that I re-activated it. 

Sorry, I thought it was the same problem than your ticket
